test/e2e/apimachinery/webhook.go

```diff
@@ -1155,6 +1157,8 @@ func testWebhook(f *framework.Framework) {
 		Labels: map[string]string{
 			skipNamespaceLabelKey: skipNamespaceLabelValue,
 			f.UniqueName:          "true",
+			// TODO: route namespace creation via framework.Framework.CreateNamespace in 1.24
+			admissionapi.EnforceLevelLabel: string(admissionapi.LevelPrivileged),
```

Comment:
should be able to make this restricted, we're not creating any pods inside this namespace, so there's no need for it to be privileged

Reply:
reset to restricted, i honestly don't remember the bump to privileged. let's see what e2e says.